api/src/db/models/opportunity_models.py

```diff
@@ -81,6 +81,11 @@ def applicant_types(self) -> list[ApplicantType]:
         # Helper method for serialization of the API response
         return [a.applicant_type for a in self.link_applicant_types]
 
+    def __repr__(self):
+        return (f"<Opportunity(opportunity_id={self.opportunity_id}, "
+                f"opportunity_number='{self.opportunity_number}', "
+                f"opportunity_title='{self.opportunity_title}', "
+                f"agency='{self.agency}', category='{self.category}', ")
```

Comment:
@chouinar - hope its ok to add repr methods so printing is a little bit cleaner 

![image](https://github.com/HHS/simpler-grants-gov/assets/93001277/5c854e79-ac76-4d81-bc97-5e1aa31401ae)